dataherald/sql_generator/__init__.py

```diff
@@ -26,6 +28,25 @@ def create_sql_query_status(
     ) -> NLQueryResponse:
         return create_sql_query_status(db, query, response)
 
+    def format_intermediate_representations(
+        self, intermediate_representation: List[Tuple[AgentAction, str]]
+    ) -> List[str]:
+        """Formats the intermediate representation into a string."""
+        formatted_intermediate_representation = []
+        for item in intermediate_representation:
+            tool = item[0].tool
+            tool_input = item[0].tool_input
+            log = item[0].log
+            thought = log.split("Action:")[0]
+            observation = item[1]
```

Comment:
You can set this values directly in the `formatted_intermediate_representation.append(` there is no need to make the assignation